### PR TITLE
Feature/355 details screen for account

### LINKED ISSUE #355: Details screen for account
As a user I'd like to easily access and manage my accounts.
 
* When the user clicks on one of the account from the "My Accounts" screen, he should see the following screen with the proper data for that particular account:
image
* When the user clicks on "Show Balance Details" it opens the "Balance drawer" (the current one is totally fine)
* When he clicks on "Manage Identity" the user gets redirected to that account "Identity screen"
* When he clicks on "Remove Account", it deletes the account

### PR DESCRIPTION
**Pull request by ahoseinian**
- Add useBalance hook
- Add Account detail screen

fixes #355

### CODE REVIEW

File: src/api/hooks/useBalance.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+export function useBalance({address}: {address: string}) {
+  const {api} = useApi();
+  const [balance, setBalance] = useState<AccountInfo | null>(null);
+
+  useEffect(() => {
+    let localUnsub: () => void | null;
+    if (api) {
+      api?.query.system
+        .account(address, (accountInfo) => {
```
Comment (silva-fj) [inline src/api/hooks/useBalance.ts]: We already have a hook for this (`useAccountInfo`)

**Review by silva-fj — COMMENTED** [review]
How do we clear identities and set sub identities now?

**Comment by ahoseinian** [comment]
> How do we clear identities and set sub identities now?

#393

File: src/api/hooks/useBalance.ts (hunk 1 above)
Comment (ahoseinian) [reply src/api/hooks/useBalance.ts]: thanks 👍  I added it on 843ce24  ✅

File: src/navigation/AppNavigator.tsx [hunk 2]
```diff
@@ -81,6 +86,14 @@ function DashboardStackNavigator() {
       <DashboardStack.Screen name={routeKeys.treasuryScreen} component={TreasuryScreen} />
       <DashboardStack.Screen name={routeKeys.submitTipScreen} component={SubmitTipScreen} />
       <DashboardStack.Screen name={routeKeys.motionsScreen} component={MotionsScreen} />
+      <DashboardStack.Screen name={routeKeys.registerSubIdentitiesScreen} component={RegisterSubIdentitiesScreen} />
```
Comment (ahoseinian) [inline src/navigation/AppNavigator.tsx:89]: @franjs  I think this would fix the #379 problem

File: src/screen/tips/TipDetailScreen.tsx [hunk 3]
```diff
@@ -128,7 +128,7 @@ function TipDetailScreen({route}: ScreenProps) {
         data={tip.tips}
         style={[globalStyles.paddedContainer, styles.container]}
         ItemSeparatorComponent={Divider}
-        renderItem={({item}) => {
+        renderItem={({item}: {item: any[]}) => {
```
Comment (silva-fj) [inline src/screen/tips/TipDetailScreen.tsx]: Let's use `FlatList` instead to get the correct types

Comment (ahoseinian) [reply src/screen/tips/TipDetailScreen.tsx]: done ✅

**Review by silva-fj — APPROVED** [review]

**Review by ishrak-imam — APPROVED** [review]

**Pull request merged**